p2p/types/network.go

```diff
@@ -1,16 +1,13 @@
 package types
 
 import (
-	"fmt"
-	"sort"
-
 	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
 	"github.com/pokt-network/pocket/shared/modules"
 )
 
 // CLEANUP(olshansky): See if we can deprecate one of these structures.
```

Comment:
🧹

Reply:
I wanted to leave the structures commented to preserve the meaning of the CLEANUP item that's going to be tended in a separate TECHDEBT PR